CMakeLists.txt

```diff
@@ -1217,6 +1217,29 @@ else ()
   set(HAVE_HALF_FLOAT OFF)
 endif ()
 
+# Define a macro to check if we have included and initialized libipt.
```

Comment:
nit: A CMake "macro" is an explicit function-like piece of code.  This is just a variable, not a macro.

Reply:
Done.